### DISCUSSION TITLE
Allow mixed frontmatter styles

### DISCUSSION THREAD

**Discussion by novellac** [tinacms/tinacms#3583] [Ideas]
## Feature request
Enable TinaCMS to read both TOML and YAML frontmatter.

## Context
I'm working with a legacy codebase where some documents' frontmatter use YAML and other documents' frontmatter use TOML. 

## Issue
While migrating from Forestry to TinaCMS, I've noticed that my TOML content doesn't show up in the TinaCMS editor. Forestry doesn't seem to have an issue with reading both TOML and YAML, but TinaCMS can read [YAML or TOML](https://tina.io/blog/tina-v-1.1.3), but not both. I don't mind saving the new/edited content as one or the other, but would like them both to be picked up so my content authors can use the TinaCMS visual editor for all content.

## Possible workarounds
I could convert the documents from one format to the other by hand, or write a script to do it.

**Comment by jeffsee55** [comment]
It seems like we could handle this for existing data, trying `yaml` first, then falling back to `toml` and storing which one as metadata. Would you have a need for specifying the format when creating new documents?

**Reply by novellac** [reply]
I have no need to specify the format when creating new documents; either YAML or TOML would be fine for me.
